src/main/java/com/example/demo/services/impl/UserServiceImpl.java

```diff
@@ -51,20 +57,38 @@ public User getUserById(final long id) {
     }
 
     @Override
-    public void addUser(final UserCreation user) {
+    public boolean addUser(final UserCreation user) {
+        if (user.getAge() < 0) {
+            log.error("age validation error");
```

Comment:
For each string in all classes create private static final string VAR_NAME = "your message "